### DISCUSSION TITLE
locus short description vs long description

### DISCUSSION THREAD

**Discussion by urapadmin** [arch-kiosk/arch-kiosk-office#1052] [users: ideas for new features] [answered]
Pompei has a short description field (limited to 60ch or so) for a context / locus different from a longer description, it seems. 
Would that make sense for Urap or other projects? One could show it in locus lists...

**Comment by luizaogs** [comment]
Huh, I think this is why the old discussions about nicknaming features came about -- because in feature and/or locus lists one does not get a description, only the type. And if you have 50 container loci that can get confusing. So this strikes me as pretty useful, actually, upon first thought (more so than nicknames, at least).

**Comment by urapadmin** [comment, marked as answer]
picked up now by #2751
